clients/drcachesim/tracer/raw2trace.h

```diff
@@ -83,23 +84,11 @@ namespace drmemtrace {
 #    define DEBUG_ASSERT(x) /* nothing */
 #endif
 
-#define OUTFILE_SUFFIX "raw"
 #ifdef BUILD_PT_POST_PROCESSOR
 #    define OUTFILE_SUFFIX_PT "raw.pt"
```

Comment:
This is part of the OUTFILE_SUFFIX* group though: it seems odd to separate it.  Should we move it to _shared.h too?

Reply:
Done.